types/superagent-no-cache/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for superagent-no-cache 0.1
+// Project: https://github.com/johntron/superagent-no-cache
+// Definitions by: Michael Ledin <https://github.com/mxl/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as request from 'superagent';
+
+declare const plugin: request.Plugin;
+
+export default plugin;
```

Comment:
See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default